configure.ac

```diff
@@ -9,8 +9,10 @@ AC_DEFUN([FAF_PARSE_WITH],
     fi
     m4_popdef([FAF_UC_PACKAGE])])
 
+m4_define([faf_version], [2.2.0])
```

Comment:
Argh, forgot that this won’t get touched during builds, so the question is: do we care about being super-specific in development builds? I don’t think it matters at all for faf_schema, but maybe someone prefers seeing the commit hash in webfaf (and I don’t mean myself).

Reply:
@ernestask Approximately how difficult would it be to make the git commit appear in the footer with the current setup? Asking for a friend.

Reply:
Tree-fiddy, approximately.

It can be baked in fairly easily, but the problem then is with tito builds, because it uses git-archive, meaning that the git metadata would not be there, requiring a custom builder. Unless you are into necromancy, which you could use to add additional mangling to the spec file. Still going to be fiddly, though.

Reply:
Gotcha.

Reply:
So do you want me to look into it? If that’s genuinely helpul to you, I can figure something out.

Reply:
I mean your friend by saying “you”, of course.

Reply:
No, not really, thanks. I was just wondering, because it was _fairly_ easy before, with autotools. At the moment, tito looks like an opaque monster to me, but I haven't started looking into it yet. I don't really need the information for anything in the UI.